locales/en.json

```diff
@@ -1,22 +1,22 @@
 {
 	"Collaborative markdown notes": "Collaborative markdown notes",
-	"Realtime collaborative markdown notes on all platforms.": "Realtime collaborative markdown notes on all platforms.",
-	"Best way to write and share your knowledge in markdown.": "Best way to write and share your knowledge in markdown.",
+	"Realtime collaborative markdown notes on all platforms.": "Collaborate on markdown notes on all platforms in realtime.",
```

Comment:
Should be `real-time` or `real time` -> https://www.merriam-webster.com/dictionary/real%20time